### ISSUE TITLE
Support Android

### ISSUE THREAD

**Issue by lex111** [garrettmac/react-native-animated-icons#1]
Is it planned?

**Comment by garrettmac** [comment]
yes, i've been busy and will be for a few weeks but I feel it should work. Have you tried?

**Comment by lex111** [comment]
I did not try, but I just copy the contents of `index.ios.js` to `index.android.js`?
